tests/e2e/e2e_test.go

```diff
@@ -111,3 +111,76 @@ func (s *IntegrationTestSuite) TestBankTokenTransfer() {
 		)
 	})
 }
+
+/*
+bank send with fees two fees.
+The following test is for the case that:
+photon fee: higher than or equal to min_gas_price in app.toml, but lower than the photon fee required in globalfee.
+stake fee: higher than stake fee required in globalfee.
+*/
+func (s *IntegrationTestSuite) TestGlobFee() {
+	s.Run("send_tokens_to_test_globalfee", func() {
```

Comment:
To be on the safe side it might be a good idea to write another test with invalid fees and expect the transaction to fail